packages/react-components/src/theming/themes.ts

```diff
@@ -87,7 +87,8 @@ const partialDarkTheme: PartialTheme = {
     neutralDark: '#f4f4f4',
     black: '#f8f8f8',
     white: '#252423',
-    whiteTranslucent40: 'rgba(0, 0, 0, 0.4)'
+    whiteTranslucent40: 'rgba(0, 0, 0, 0.4)',
+    redDark: '#f1707b'
```

Comment:
Does this affect calling ui elements as well?

![image](https://github.com/Azure/communication-ui-library/assets/73612854/15d236ad-ee73-4b57-b1cf-8ce29b6ff596)

Reply:
Yeah, good call. Let me check